compiler/passes/src/loop_unrolling/unroll_expression.rs

```diff
@@ -19,5 +19,13 @@ use leo_ast::*;
 use crate::Unroller;
 
 impl ExpressionReconstructor for Unroller<'_> {
-    type AdditionalOutput = ();
+    type AdditionalOutput = bool;
```

Comment:
Where is the additional output being used?
I image to eliminate dead code but where is it being calculated

Reply:
It's being calculated in `unroll_statement/reconstruct_definition()` to remove constant declarations from the AST.

Reply:
Are there any spots where the additional output is `true` or is this a placeholder?

Reply:
Resolving due to filtering on L37 of `unroll_statement.rs`.